jdat_notebooks/background_estimation_imaging/Imaging Sky Background Estimation.ipynb

```diff
@@ -40,7 +40,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    sources['masked'] = np.zeros(len(sources), dtype=bool)
</pre><p> </p><p>(numpy raises a deprecation warning: DeprecationWarning: <code>np.bool</code> is a deprecated alias for the builtin <code>bool</code>. To silence this warning, use <code>bool</code> by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use <code>np.bool_</code> here.</p><p>Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/dat_pyinthesky/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>